### PR TITLE
Stats page improvements

### PR DESCRIPTION
**Pull request by kolebjak**

### CODE REVIEW

File: src/sections/stats/components/RecentTradesTable/RecentTradesTable.utils.tsx [hunk 1]
```diff
@@ -31,154 +41,165 @@ export const useRecentTradesTable = (data: TRecentTradesTableData) => {
...
-        id: "totalValue",
-        header: t("stats.overview.table.trades.header.totalValue"),
-        sortingFn: (a, b) =>
-          a.original.totalValue.gt(b.original.totalValue) ? 1 : -1,
-        cell: ({ row }) =>
-          isDesktop ? (
-            <Text tAlign="center" color="white">
-              <DisplayValue value={row.original.totalValue} isUSD />
-            </Text>
+  const spot = useSpotPrices(
```
Comment (vkulinich-cl) [inline src/sections/stats/components/RecentTradesTable/RecentTradesTable.utils.tsx:49]: Could you do those calculations in the file where you prepare the data for the table, including the spot price and hydra account address?

File: src/api/spotPrice.ts [hunk 2]
```diff
@@ -32,6 +32,8 @@ export const useSpotPrices = (
   const assets = assetsIn
     .filter((a): a is u32 | string => !!a)
     .map((a) => a.toString())
+    .filter((v, i, a) => a.indexOf(v) === i)
```
Comment (vkulinich-cl) [inline src/api/spotPrice.ts:35]: what is for?

Comment (vkulinich-cl) [reply src/api/spotPrice.ts:35]: okay, could we use a new Map instead? should remove duplicates and convert to string, I assume

Comment (kolebjak) [reply src/api/spotPrice.ts:35]: I think Set would be better if we need unique values, right?

Comment (vkulinich-cl) [reply src/api/spotPrice.ts:35]: yes yes

File: src/api/spotPrice.ts [hunk 3]
```diff
@@ -29,13 +29,14 @@ export const useSpotPrices = (
 ) => {
   const { tradeRouter } = useRpcProvider()
 
-  const assets = assetsIn
-    .filter((a): a is u32 | string => !!a)
-    .map((a) => a.toString())
+  const assets = new Set(
```
Comment (vkulinich-cl) [inline src/api/spotPrice.ts:32]: I have just tested, that Set doesn't remove duplicates. But Map does

Comment (kolebjak) [reply src/api/spotPrice.ts:32]: There can't be duplicate values in Set

**Review by vkulinich-cl — APPROVED** [review]

**Pull request merged**